src/ngx_lua_resty_lmdb_module.c

```diff
@@ -380,6 +533,28 @@ static ngx_int_t ngx_lua_resty_lmdb_init(ngx_cycle_t *cycle)
         return NGX_ERROR;
     }
 
+    /* check lmdb validation tag */
+
+    if (ngx_lua_resty_lmdb_validate(cycle, lcf) != NGX_OK) {
+        ngx_lua_resty_lmdb_close_file(cycle, lcf);
```

Comment:
A log log should exist here indicating the database is being removed. However, it should not be "LMDB database is corrupted or incompatible" as that is a different condition than tag validation failure. Consider fixing the error messages.

Reply:
done